src/script/redmine-upkeep.py

```diff
@@ -39,6 +40,13 @@
     pass
 GITHUB_TOKEN = os.getenv("GITHUB_TOKEN", GITHUB_TOKEN)
 
+GITHUB_ACTIONS = os.getenv("GITHUB_ACTIONS") == 'true'
+GITHUB_SERVER_URL = os.getenv("GITHUB_SERVER_URL", "https://github.com/")
+GITHUB_REPOSITORY = os.getenv("GITHUB_REPOSITORY", "ceph/ceph")
+GITHUB_RUN_ID = os.getenv("GITHUB_RUN_ID", "nil")
```

Comment:
Perhaps we could use this info to conditionally skip this on non ceph/ceph forks? Many folks complain that when they fork Ceph, lots of CI checks fail because of missing API keys (yes, they should disable workflows on forked repos). Just thinking aloud.